cns/client.go

```diff
@@ -35,6 +35,7 @@ const (
 
 const (
 	ReleaseVSAN67u3 = "vSAN 6.7U3"
+	ReleaseVSAN70   = "7.0"
```

Comment:
for 7.0 patch release, what is the release version?

Reply:
for 7.0U1 it will be 7.1